### PR TITLE
Gitc 652 add ability for grant owner to see all their grants under the me menu including pending approval grants

### PR DESCRIPTION
**Pull request by schultztimothy**
##### Description

This adds the ability for grant owner to see all their grants under the me menu including "pending approval" grants. Within grant card helper text and ui changes indicate why and that the grants are currently not visible.

##### Refers/Fixes

Gitc 652

https://user-images.githubusercontent.com/6887938/149600925-ad0a7870-a20b-42cd-8c61-025c957ab0d1.mov

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

File: app/grants/views.py [hunk 1]
```diff
@@ -686,7 +689,10 @@ def get_grants_by_filters(
     three_months_ago = timezone.now() - timezone.timedelta(days=90)
 
     # 1. Filter grants by network and hidden = false
-    _grants = Grant.objects.filter(network=network, hidden=False)
+    _grants = Grant.objects.filter(network=network)
+    
+    if not hidden_grants:
+        _grants = Grant.objects.filter(hidden=False)
```
Comment (chibie) [inline app/grants/views.py:695]: shouldn't this be

```
    if not hidden_grants:
        _grants = _grants.filter(hidden=False)
 ```
 
 to keep the network filter + hit db once

**Review by chibie — CHANGES_REQUESTED** [review]
looks good man... just one fix

File: app/grants/views.py (hunk 1 above)
Comment (schultztimothy) [reply app/grants/views.py]: Thanks for catching this @chibie!

**Review by kevinrolsen — APPROVED** [review]

**Review by chibie — APPROVED** [review]

**Review by gdixon — APPROVED** [review]
LGTM and works great! Nice work @schultztimothy 🚀

**Pull request merged**